addon/webextension/site-helper.js

```diff
@@ -7,7 +7,7 @@ function sendEvent(type, detail) {
 }
 
 document.addEventListener("has-saved-shot", (event) => {
-  chrome.runtime.sendMessage({
+  browser.runtime.sendMessage({
```

Comment:
Any reason you skipped the Promise conversion for these `browser.runtime.sendMessage` calls?